jha/src/views/enrolment/SuppBenInfoPage.vue

```diff
@@ -63,7 +63,7 @@
             </div>
 
             <div v-if="onlySuppBen">
-              <p class="mt-4 mb-1 font-weight-bolder">Do you have any children on your Medical Services Plan account?</p>
+              <p class="mt-4 mb-1 font-weight-bolder">Do you have any children on your Medical Services Plan (MSP) account?</p>
```

Comment:
Based from the other text changes, I'm not sure if it's only supposed to be acronym (just "MSP") or "Medical Services Plan" needs to be spelled out as well.

Reply:
Because if they're applying for SuppBen only, this will be the first time they see Medical Services Plan, so it should be spelled out